ectweitmann/book-club-ui
# Feature/displaying favorites ect

## What Features are Being Added?
- Add functionality to the favorites view
- Add links to return a user to the home page
- Create component to handle bad paths called `Error404`
- Add a message to inform a user they haven't selected any favorites
- Add logo in place of h1
 - Add alt text to image covers
 
## What is Being Refactored?
- Refactored tabbing of `BookContainer`

## Where should the reviewer start?
- Multiple changes were made throughout all component files.
- Start in App to trace the changes to the Route paths

## How should this be manually tested?
- `git fetch` and npm start and navigate the app. All functionality should be up and running (favoriting, learn more button, error handling etc.)

## What are the relevant tickets?
- MVP app functionality for favoriting.

## Any Other Notes/Questions/Problems?
- Would like to go back and add responsive styling if possible.
